doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+    So typically the only sources that use the PSF template as a model are the very faintest sources.
+
+.. _re-apportion:
+
+Re-Apportioning Flux
+^^^^^^^^^^^^^^^^^^^^
+
+This is where the actual "deblending" takes place.
+Once a symmetric or PSF template has been made for each source they are all combined into a single model for the entire blend.
+For each source the ratio of its template model to the full model is made, and that ratio is multiplied by the observed image to generate the final model for the source.
+
+.. note::
+    In practice it is a bit more complicated than this.
+    Accounting for stray flux and flux at the edges of an image also needs to be included,
+    but this general outline gives a good overview of the dominant features of the single-band model.
+
+This operation (also called flux-conservation) ensures that all of the flux, including the noise in the image, is contained in one (and only one) source in proportion to the fraction of flux from its template to the full model for each pixel.
+Looking at the four brightest sources in the image we see the re-apportioned models below:
+
+.. image:: images/singleConserved0.png
+.. image:: images/singleConserved1.png
+.. image:: images/singleConserved2.png
+.. image:: images/singleConserved3.png
+
+This example was chosen because it is a sufficiently complicated blend and one that the single band deblender struggles with.
+Most blends have only 2 or 3 faint sources and the single-band deblender does a reasonable job deblending them.
+For example, had source 3 been absent from the image
+
+.. image:: images/simpleBlend.png
+
+the entire blend would have been modeled better:
+
+.. image:: images/simple0.png
+.. image:: images/simple1.png
+.. image:: images/simple2.png
+
+.. note::
+    Source 3 was subtracted using the model obtained in `multi-band_deblending`_.
+
+But even this model can be improved.
+While source 0 is all of the flux in its footprint (since no other model overlaps with it), there is still some mixing between sources 1 and 2.
+This is partly due to one or two undetected sources in source 2, but also because symmetric templates alone are not enough to separate objects in a monochromatic image.
+A monotonic projection can help, but looking back at the full color image we see that the easiest way for our eyes to separate the sources is by using color information, which will be discussed in the next section.
+
+See `Lupton 2001 <https://www.astro.princeton.edu/~rhl/photomisc/deblender.pdf>`_ for more details on the SDSS implementation of this algorithm.
+
+.. _multi-band_deblending:
+
+Multi-band Deblending with scarlet (lite)
+-----------------------------------------
+
+The difference between scarlet and scarlet lite
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The science pipelines currently use models generated by the "lite" version of *scarlet*, so it's worth understanding the difference between the two versions.
```

Comment:
Scarlet lite doesn't have any documentation yet. In the coming months it will be its own stand alone package with additional tests and docs, but for these _are_ the scarlet lite docs.